docs/package.json

```diff
@@ -5,8 +5,9 @@
   "license": "MIT",
   "scripts": {
     "dev": "next -p 8000",
-    "build": "next build",
+    "build": "next build && yarn sitemap",
     "start": "next start -p 8000",
+    "sitemap": "yarn next-sitemap --config lib/next-sitemap.ts",
```

Comment:
Built after Vercel runs `next build`, confirmed the `sitemap.xml` and `robots.txt` get generated and are accessible from the preview URL.